### DISCUSSION TITLE
Copy/Propagate access/creation time and thumbnail image of synced bookmarks

### DISCUSSION THREAD

**Discussion by hochl** [floccusaddon/floccus#1401] [Ideas]
I have tons of bookmarks on many different computers. I have created a sync bookmark folder on two of these computers to try out floccus. When I drag a bookmark into the sync folder on computer A, computer B will receive the bookmark, but with last access and creation timestamps of the current time, most probably because they were just created on machine B. For various reasons, I would love to keep the original access and creation time of a copied bookmark. Would it be possible to initialize the access and creation date of the newly created bookmark on machine B from the one on machine A? It could be an option to enable this behavior, but for various reasons, including bookkeeping, it is quite important to know when I added a bookmark and when I last accessed it. It would be pretty cool if a change to these times would also propagate to all other machines, in case I load a synced bookmark on one machine. Ideally the thumbnail image of the link would also get copied. Or is that all not possible for some technical reason?

P.S.: I am running rclone as WebDAV server on a RPi.

**Comment by marcelklehr** [comment]
Hi  @hochl 

Thank you for submitting your idea!

> Or is that all not possible for some technical reason?

It is indeed not possible for technical reasons -- the web extension APIs that floccus uses do not allow setting a custom creation time nor do they allow accessing bookmarks' thumbnails :(
